src/main/java/com/wanderersoftherift/wotr/core/rift/RiftLevelManager.java

```diff
@@ -35,33 +36,86 @@
 import net.neoforged.neoforge.network.PacketDistributor;
 import net.neoforged.neoforge.server.ServerLifecycleHooks;
 import org.apache.commons.io.FileUtils;
+import org.jetbrains.annotations.Nullable;
 
-import javax.annotation.Nullable;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.util.List;
 import java.util.Optional;
 import java.util.Random;
+import java.util.Set;
 
-public class RiftLevelManager {
+/**
+ * Static manager for handing access to, creation, and destruction of a rift
+ */
+public final class RiftLevelManager {
+
+    private RiftLevelManager() {
+    }
 
     /**
      * @param id
      * @return Whether a level with the given id exists
      */
     public static boolean levelExists(ResourceLocation id) {
         var server = ServerLifecycleHooks.getCurrentServer();
-        var existingRift = server.forgeGetWorldMap().get(ResourceKey.create(Registries.DIMENSION, id));
-        return existingRift != null;
+        var dimension = server.forgeGetWorldMap().get(ResourceKey.create(Registries.DIMENSION, id));
```

Comment:
Can the WorldMap contain null for the ID? Else we can just use a contains in the location.

Reply:
Switched to a contains check. Seems to work fine from testing.